modules/extension/transform/pom.xml

```diff
@@ -66,18 +66,23 @@
 
     <dependency>
       <groupId>org.geotools</groupId>
-      <artifactId>gt-property</artifactId>
+      <artifactId>gt-cql</artifactId>
       <version>${project.version}</version>
-      <scope>test</scope>
     </dependency>
 
     <dependency>
       <groupId>org.geotools</groupId>
-      <artifactId>gt-cql</artifactId>
+      <artifactId>gt-process-feature</artifactId>
       <version>${project.version}</version>
-      <scope>test</scope>
+      <scope>compile</scope>
```

Comment:
Why the extra scope here? And does the module really need to depend on process-feature, I don't understand why?